New package: EDM4hep v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: EDM4hep
- Repository: https://github.com/peremato/EDM4hep.jl
- Created by: @peremato
- Version: v0.1.0
- Commit: b43270b9244a954fe6fb18da6c95d1fc0c0e2b2e
- Reviewed by: @peremato
- Reference: https://github.com/peremato/EDM4hep.jl/commit/b43270b9244a954fe6fb18da6c95d1fc0c0e2b2e#commitcomment-138571020
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd627752f47924a10dec82834e4e48c4f9a14fd4dab80c4b0568aba79dd154fe5c115f1640bc9c4fbd7ad337c3242cd4f049f86295bcf1ec9edb0e6fc15b7131eef71d92c2e56919ad76414248a077945461aca4a59b120312db33c904dd69062a6d7107f2c1ee40c5de0db4ff5d6317aca1fca17f90d52c2bb04f727c66d1114c4144b48663eeaadd99ec2fc4cba2e422c2f85aa0939a06b66a9e813bfa09c67e5b334298439544c6f9ae3c236f82f3c9a -->

goerz commented:
Thank you for submitting your package! Apart from the technical issues with the missing `compat` specifications, a better name for this package would probably be `HEPEventDataModel` or `EventDataModel4HEP` or something like that.

peremato commented:
Dear [Michel](@goerz). Thanks for your suggestion. The problem is that this package is a counterpart of a C++ based package with the same name. Keeping the name indicates that it is basically the same functionality and compatible with the data files produced by the C++ one.

JuliaRegistrator committed:
New package: EDM4hep v0.1.0

UUID: eb32b910-dde9-4347-8fce-cd6be3498f0c
Repo: https://github.com/peremato/EDM4hep.jl.git
Tree: 301d1d2a7e30dba553a53be82944371fc73881b5

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

goerz commented:
[noblock] ok, that sounds reasonable. You’ll want to edit your comment to contain `[noblock]`
